README.md

```diff
@@ -850,6 +850,7 @@ Table of Contents
   * [configure.it](https://www.configure.it/) — Mobile app development platform, free for 2 projects, limited features but no resource limits
   * [codenameone.com](https://www.codenameone.com/) — Open source, cross platform, mobile app development toolchain for Java/Kotlin developers. Free for commercial use with unlimited number of projects
   * [Deta](https://www.deta.sh) – Deploy unlimited number of Node.js and Python apps for free. Includes free DBs, Auth and email.
+  * [Deno Deploy](https://deno.com/deploy/) — Serverless platform based on Deno. Deployed to 25 Google Cloud regions globally. 50ms/request CPU time and 100,000 requests/day with a deno.dev subdomain, while in beta.
```

Comment:
```During the initial public beta for Deno Deploy the service is free to use. Once Deno Deploy reaches General Availablity, detailed pricing will be announced.```

unfortunately we cant add this one, it'll be stale as people forget to update the list and there is no indication from them that a ongoing free tier is planned.  As per top of README, free during beta is not applicable.

Maybe i missed such a statement though

Reply:
Sure, we can draft this for now.